packages/playground/src/explorer/utils/helpers.ts

```diff
@@ -137,3 +151,23 @@ export const getQueries = (mixedFilters: MixedFilter): Partial<NodesQuery> => {
   }
   return options;
 };
+
+export async function getFarms(queries: Partial<FarmsQuery>): Promise<Pagination<Farm[]>> {
+  try {
+    const farms = await gridProxyClient.farms.list(queries);
+    return farms;
+  } catch (error) {
+    console.error("An error occurred while requesting farms:", error);
+    throw error;
+  }
+}
+
+export async function getTwins(queries: Partial<TwinsQuery> = {}): Promise<Pagination<Twin[]>> {
```

Comment:
This function is created to get the twin id of a specific farm. Can we change its name to getTwinId and edit the return value to the first object in the list?